plugins/janus_videoroom.c

```diff
@@ -441,6 +442,11 @@ typedef struct janus_videoroom_rtp_relay_packet {
 
 
 /* Freeing stuff */
+static void janus_videoroom_subscriber_dereference(janus_videoroom_subscriber *s) {
+	if(s)
```

Comment:
IMHO I'd replace the test with an assert.  When I saw it it made me wonder how it would be possible.  I'm all for defensive programming, but being overly defensive can make one a bit confused :-)